js/index.js

```diff
@@ -62,6 +65,15 @@ function handlePostalCodeInput(codePostal, next) {
 	request.send()
 }
 
+/*
+contrat_de_travail <select> value can trigger the display
+of the heures_remunerees_volume <input> field.
+*/
+function handleTempsPartielSelect(contrat, next) {
+	document.querySelector('#temps_partiel_container').hidden = contrat === 'temps_plein'
```

Comment:
Beware, by using the `hidden` API you're dropping [IE10 compatibility](http://caniuse.com/#feat=hidden) and at least need to update the [README](https://github.com/sgmap/cout-embauche#also-known-to-be-compatible-with).

Workaround is to use `.setAttribute('hidden', contrat === 'temps_plein' || '')`.

> Or you can use a polyfill or write it yourself by extending `Element` :wink: